test/playlist.test.js

```diff
@@ -385,11 +385,92 @@ function(assert) {
 
   assert.equal(seekable.start(0), 0, 'starts at the earliest available segment');
   assert.equal(seekable.end(0),
-              9 - (2 + 2 + 1),
-              'allows seeking no further than three segments from the end');
+              9 - (2 + 2 + 1 + 2),
```

Comment:
It's slightly unclear what's going on here.  It might make sense to make this wordier so that the test is a bit quicker to understand.  I'm guessing something like `playlists.segments[0].duration + segments[1].duration...`.  I'm also guessing the 9 is the total sum of the duration.  Either that or a comment to highlight what these numbers are would be helpful.

Reply:
Also I can't comment on code you haven't touched :) But it might be wise to change the `let playlist` to a `const playlist` unless you're expected the functions it's passed into to reassign it in some way.